fili-core/src/test/groovy/com/yahoo/bard/webservice/util/TableUtilsSpec.groovy

```diff
@@ -87,19 +82,6 @@ class TableUtilsSpec extends  Specification {
         query.dependentFieldNames >> ([metric1, metric2, metric3] as Set)
 
         expect:
-        TableUtils.getColumnNames(request, query, new PhysicalTable("", DefaultTimeGrain.DAY.buildZonedTimeGrain(DateTimeZone.UTC), [:])) == [d1Name, metric1, metric2, metric3] as Set
-    }
-
-    def "logicalName to physicalName mapping not required for metrics" () {
```

Comment:
We are not converting logicalName to physicalName for both dimension and metrics now, originally dimensions were. So testing metric do not get converted into physicalName loses its meaning I think.